pkg/registry/image.go

```diff
@@ -144,3 +144,21 @@ func (ImageCtrl) getExposedPortsFromCfg(cfg *containerv1.ConfigFile) []Port {
 	}
 	return result
 }
+
+func GetDevTagFromGlobal(image string) string {
+	if !strings.HasPrefix(image, constants.GlobalRegistry) {
+		return ""
+	}
+
+	// separate image reference and tag eg: okteto.dev/image:tag
+	reference, _, found := strings.Cut(image, ":")
```

Comment:
I guess it's ok because i don't think it can happen but if the image is in the sha format there is no `:` and this will fail

Reply:
i changed this to check sha format first and then tag.